src/common/chat-participants/powerpages/PowerPagesChatParticipantUtils.ts

```diff
@@ -16,3 +20,29 @@ export async function getEndpoint(
     }
     return cachedEndpoint;
 }
+
+/*
+    * Get component info for the active file
+    * @returns componentInfo - Entity details for active file (form or list)
+*/
+export async function getComponentInfo(telemetry: ITelemetry, orgUrl: string | undefined, activeFileParams: IActiveFileParams): Promise<string[]> {
+
+    let metadataInfo = { entityName: '', formName: '' };
+    let componentInfo: string[] = [];
+
+    if (activeFileParams.dataverseEntity == ADX_ENTITYFORM || activeFileParams.dataverseEntity == ADX_ENTITYLIST) {
```

Comment:
lets move this to a utility function - that will help when we expand to other entities